### PR TITLE
feat #91439: WBO show list and initial data

### PR DESCRIPTION
**Pull request by olsoybakk**
## Checklist
- [x] I have run linter.
- [ ] I added some tests. (to be defined further) [optional]
- [ ] I have updated [the wiki.](https://github.com/equinor/echo-maps/wiki) [optional]

## Primary description
- Shows list with icons and data
- Shows some bonus-data when user click on a wbo-feature
- [AB#91439](https://dev.azure.com/EquinorASA/13429306-9600-4820-ac5c-3e7d26fa3cb9/_workitems/edit/91439)

## Screenshots
image
image

### CODE REVIEW

File: src/layers/external/wbo/ui/WBOSheet.tsx [hunk 1]
```diff
@@ -1,22 +1,228 @@
...
+                            ([key, value]) => {
+                                return (
+                                    <span
+                                        key={key}
+                                        style={{ display: 'block' }}
+                                    >
+                                        <code style={{ fontWeight: 'bold' }}>
+                                            {key}:{' '}
+                                        </code>
+                                        <code title={value}>{value}</code>
```
Comment (ErlendAHall) [inline src/layers/external/wbo/ui/WBOSheet.tsx:96]: code element is for displaying source code in markup, did you mean to create a description list here?

File: src/layers/external/wbo/ui/WBOSheet.tsx [hunk 2]
```diff
@@ -1,22 +1,228 @@
...
+const StyledListItem = styled(ListItem)`
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+`;
+
+const StyledSection = styled.section`
+    padding: 1rem;
+    overflow-y: auto;
+`;
+
```
Comment (ErlendAHall) [inline src/layers/external/wbo/ui/WBOSheet.tsx:225]: This should be CSS. We're retiring styled components.

Comment (olsoybakk) [reply src/layers/external/wbo/ui/WBOSheet.tsx:225]: aaah, it was a copy-paste from another layer

File: src/layers/external/wbo/ui/WBOSheet.tsx (hunk 1 above)
Comment (olsoybakk) [reply src/layers/external/wbo/ui/WBOSheet.tsx:95]: this is just the beginning so other developers can create the content

**Review by ErlendAHall — APPROVED** [review]

**Pull request merged**